wheel/python/chia_rs/chia_rs.pyi

```diff
@@ -211,6 +212,9 @@ class PrivateKey:
     def sign_g2(self, msg: bytes, dst: bytes) -> G2Element: ...
     def get_g1(self) -> G1Element: ...
     def __str__(self) -> str: ...
+    def public_key(self) -> G1Element: ...
```

Comment:
Can we get rid of `get_g1` then?

Reply:
`chia-blockchain` still uses that name. if we want to transition all of them, I'm find with removing `get_g1()`